src/app/component/Button/utils.js

```diff
@@ -1,21 +1,3 @@
-export const colors = {
-  primary: {
-    background: '#140D60'
-  },
-  secondary: {
-    background: '#D31409',
-    border: ' #D31409'
-  },
-
-  tertiary: {
-    background: '#F3D3EE'
-  },
-  disabled: {
-    border: '#e5e5e5',
-    color: 'rgba(0, 0, 0, 0.2)'
-  }
-};
-
```

Comment:
Moved colors out to constants file so that they can be used in other componnets.